data/net/minecraft/world/level/levelgen/feature/HugeFungusConfiguration.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/world/level/levelgen/feature/HugeFungusConfiguration
+	METHOD <init> (Lnet/minecraft/world/level/block/state/BlockState;Lnet/minecraft/world/level/block/state/BlockState;Lnet/minecraft/world/level/block/state/BlockState;Lnet/minecraft/world/level/block/state/BlockState;Z)V
+		ARG 1 validBaseState
+		ARG 2 stemState
+		ARG 3 hatState
+		ARG 4 decorState
```

Comment:
If this is the field name, eh. But otherwise `decorationState`

Reply:
field name